vignettes/datatable-importing.Rmd

```diff
@@ -138,7 +138,7 @@ The option mechanism in R is _global_. Meaning that if a user sets a `data.table
 
 If you face any problems in creating a package that uses data.table, please confirm that the problem is reproducible in a clean R session using the R console: `R CMD check package.name`.
 
-Some of the most common issues developers are facing are usually related to helper tools that are meant to automate some package development tasks, for example, using `roxygen` to generate your `NAMESPACE` file from metadata in the R code files. Others are related to helpers that build and check the package. Unfortunately, these helpers sometimes have unintended/hidden side effects which can obscure the source of your troubles. As such, be sure to double check using R console (run R on the command line) and ensure the import is defined in the `DESCRIPTION` and `NAMESPACE` files following the [instructions](#DESCRIPTION) [above](#NAMESPACE).
+Some of the most common issues developers are facing are usually related to helper tools that are meant to automate some package development tasks, for example, using `roxygen` to generate your `NAMESPACE` file from metadata in the R code files. Others are related to helpers that build and check the package. Unfortunately, these helpers sometimes have unintended/hidden side effects which can obscure the source of your troubles. As such, be sure to double check using R console (run R on the command line) and ensure the import is defined in the `DESCRIPTION` and `NAMESPACE` files following the [instructions](#description-file-description) [above](#namespace-file-namespace).
```

Comment:
These sections were intended to have a certain class name:
https://github.com/Rdatatable/data.table/blob/ca904cb4c87257b38540b7c7cc24da802e44241a/vignettes/datatable-importing.Rmd#L30
https://github.com/Rdatatable/data.table/blob/ca904cb4c87257b38540b7c7cc24da802e44241a/vignettes/datatable-importing.Rmd#L34

It doesn't really work because the correct syntax would have been `{#DESCRIPTION}` and `{#NAMESPACE}`, respectively. It is important to have fixed `id` values for headings that are linked to because the auto-generated `id`s will change as a result of translation.

Reply:
seconded -- the `id`s are also better than the silly auto-generated names.

PS to clarify since I misread this comment: we need to fix the IDs to be `{#DESCRIPTION}`, not `{DESCRIPTION}`; the linking form `(#DESCRIPTION)` is fine:

https://www.markdownguide.org/extended-syntax/#heading-ids